Automattic/wp-calypso
# Sidebar: Remove payment processing nudge

#### Changes proposed in this Pull Request

* Confirmed with Yoav that this feature can be removed from the sidebar. It's part of a feature that didn't launch: p1578419640017200-slack-dotcom-manage-design

Before and after:
<img width="1128" alt="Screen Shot 2020-01-09 at 1 24 30 PM" src="https://user-images.githubusercontent.com/349751/72106008-6fa0de00-32e3-11ea-94ed-2e4042c35f6f.png">


#### Testing instructions

* Switch to this PR
* Make sure nothing breaks as a result of the removed code.
